yamoveset/panels/main.py

```diff
@@ -40,6 +41,8 @@ def __init__(self, parent):
         self.save.Disable()
         self.paste = wx.Button(self, wx.ID_PASTE, "Paste")
         self.paste.Disable()
+        self.add = wx.Button(self, wx.ID_ADD, "Add")

```

Comment:
Going to need to add this to the right click context menu as well.

Reply:
Also, you're not really adding an empty entry.  Add Copy might be a better name for this.

Reply:
thought about this, but that will make "add" appear when a user select any entry, even though the function will only add to the end of the list anyway so it felt redundant

Reply:
True, thinking about it, I would only add at the end of the BAC entries anyway.

Reply:
Still insisting on making it say "Add Copy" since thats what I called it in other tools.